charts/loki-distributed/values.yaml

```diff
@@ -103,6 +103,10 @@ loki:
     frontend:
       log_queries_longer_than: 5s
       compress_responses: true
+      
+    compactor:
+      working_directory: /var/loki/boltdb-shipper-compactor
+      shared_store: filesystem
```

Comment:
This should not go into the default values file. The defaults are not configured for boltdb-shipper. As far as I understand, boltdb-shipper requires a shared object store. Using it with `filesystem` doesn't seem to make sense to me. Since you should be able to run the chart with defaults, boltdb is used out of the box, which you will most probably want to change for real-world scenarios. That's also why the compactor is disabled by default.

At some point, I should probably add other config examples to the readme instead.